src/source/examples/post-campaign-v5-example.rst

```diff
@@ -0,0 +1,37 @@
+.. code-block:: javascript
+
+    POST /campaign
+    Accept: application/sellside.campaign-v5+json, application/json
+    Content-Type: application/sellside.campaign-v5+json; charset=UTF-8
+
+    {
+        "status": "ACTIVE",
+        "budgets": {
+            "daily": {
+                "limitMicros": "10000000",
+            },
+            "total": {
+                "limitMicros": "UNLIMITED",
+            }
+        }
+    }
+
+    200 OK HTTP/1.1
+    Content-Type: application/sellside.campaign-v5+json; charset=UTF-8
```

Comment:
where do we draw the line with respect to v5-ing API endpoints? :) this jumped quite some versions, I'm just curious.... Is the idea that all related endpoints are v5?

Reply:
yes, we chose for this to make it easier for the partners to understand. I know this is not according to what we usually do, but we also had some 'invisible' versions on some endpoints just for console...

We should really get some good discussions going on API versioning :D